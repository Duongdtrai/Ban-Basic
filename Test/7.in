11
15
13 628 501 378 742 896 589 338 640 136 916 556 745 84 800 
47
311 714 116 935 91 646 323 136 283 527 144 425 147 423 665 397 183 913 942 576 994 386 900 633 123 847 8 773 600 720 955 476 236 65 240 822 771 154 254 877 840 883 884 74 587 12 549 
39
724 785 145 657 776 420 256 261 258 356 254 312 723 818 43 978 328 82 421 74 941 49 968 913 533 503 298 170 409 769 263 106 620 838 757 317 784 283 594 
15
314 642 753 934 872 400 990 221 33 150 431 705 644 381 616 
41
231 116 934 147 333 836 317 730 638 598 530 933 580 458 998 669 731 467 669 771 63 364 598 777 602 679 490 746 422 600 533 729 164 323 608 96 393 314 783 595 733 
8
550 512 343 159 135 919 854 603 
42
527 726 637 165 992 892 511 334 542 670 649 934 541 634 197 277 304 587 745 781 328 813 460 265 890 13 859 849 752 997 978 289 341 449 258 546 634 763 481 31 853 7 
12
45 14 426 704 174 660 358 740 478 285 550 63 
6
627 592 941 995 186 70 
34
584 452 96 483 191 842 490 584 393 349 99 775 281 952 390 252 77 155 271 402 436 495 768 706 56 261 810 298 807 508 836 717 579 340 
43
937 177 636 962 381 703 816 606 597 88 916 756 837 43 96 659 674 320 670 816 557 617 355 575 275 856 353 474 180 937 785 107 666 841 982 470 312 316 183 883 858 705 933 
