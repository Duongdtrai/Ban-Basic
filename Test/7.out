178596240
39974254208
32329538230
670621140
24650245288
12452350
26429463160
45127320
10312302
510506316
69426106110
